### ISSUE TITLE
Indentation rule clashes with ArgumentListWrapping

### ISSUE THREAD

**Issue by zayo** [detekt/detekt#3533]
Hi,

I found out that both [Indentation] and [ArgumentListWrapping] is doing the same thing.
Following code:
```kotlin
    private val remoteUrl: String
        get() = context.getString(R.string.faq_url,
            context.getString(R.string.action_id),
            context.getString(R.string.element_id),
            context.getString(R.string.product_id),
            language)
```
is reported by both. I would expect that Indentation is silent, as this is continuation indentation of 4.

I've also tried to put everything on separate line and indent by 8 spaces. Which was again reported by both, while I would expect this to be reported only from Indentation, as the wrap was fulfilled.

If I'm not mistaken, the ArgumentListWrapping is just duplication with less options, that could be dropped. Or making both rules taking a smaller scope as they are likely doing the same thing.

* Version of detekt used: 1.16.0-RC3

**Comment by chao2zhang** [comment]
Hello @zayo, thank you for the bug report. 
Did you try the following format?
```kotlin
    private val remoteUrl: String
        get() = context.getString(
            R.string.faq_url,
            context.getString(R.string.action_id),
            context.getString(R.string.element_id),
            context.getString(R.string.product_id),
            language
       )
```

Meanwhile, [Indentation](https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/IndentationRule.kt) and [ArgumentListWrapping](https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/ArgumentListWrappingRule.kt) are both implemented by ktlint. If we cannot find a solution, we should report an issue there.

**Comment by zayo** [comment]
Hello @chao2zhang,

yes, that's what I did to fix it. The problem is, that it's the only possible way how to approach it.
Let's say, I want: 
```yml
Indentation:
    active: true
    autoCorrect: true
    indentSize: 4
    continuationIndentSize: 8
```
The Indentation rule would work, but this would be reported by ArgumentListWrapping rule as non compliant code because of the number of spaces.

Based on what you've said, it's sounds better to just drop/disable ArgumentListWrapping, since the Indentation does the same with more options. But I've just tested it again, and the `continuationIndentSize` param is not taken into account.

**Comment by chao2zhang** [comment]
Good call. I checked ktlint's implementation and I don't see any code reading `continuationIndentSize`. I will remove this option from detekt's wrapper.

**Comment by schalkms** [comment]
@chao2zhang is there anything left to do to close this issue?

**Comment by chao2zhang** [comment]
We have mentioned this in https://github.com/detekt/detekt/discussions/3784, so I believe there is nothing actionable at this moment.
